java-api-core/src/com/kinvey/java/store/UserStoreRequestManager.java

```diff
@@ -724,7 +724,7 @@ public LoginToTempURL MICLoginToTempURL(String username, String password, String
         data.put("password", password);
 
         HttpContent content = new UrlEncodedContent(data);
-        LoginToTempURL loginTemp = new LoginToTempURL(this, tempURL, clientId, content);
+        LoginToTempURL loginTemp = new LoginToTempURL(this, clientId, tempURL, content);
         loginTemp.setRequireAppCredentials(true);
```

Comment:
Does this change have to do with the TLS fix?

Reply:
No, It's improvement for MIC. Do you think it should be filed as separate ticket(PR)?

Reply:
Yes I think that is best. I created `MLIBZ-2576` for this, please move this change to a branch for that ticket.